web/client/components/I18N/Number.jsx

```diff
@@ -8,29 +8,22 @@
 const PropTypes = require('prop-types');
 const React = require('react');
 const {FormattedNumber} = require('react-intl');
-const { checkRounding } = require('../../utils/CoordinatesUtils');
 class NumberFormat extends React.Component {
     static propTypes = {
-        value: PropTypes.oneOf([PropTypes.object, PropTypes.number]),
-        numberParams: PropTypes.object,
-        roundingBehaviour: PropTypes.string
+        value: PropTypes.oneOfType([PropTypes.object, PropTypes.number]),
+        numberParams: PropTypes.object
     };
 
     static contextTypes = {
         intl: PropTypes.object
     };
 
     static defaultProps = {
-        value: new Date(),
-        roundingBehaviour: "round"
```

Comment:
So, this is a better fix than adding

numberParams: {}

here?

Reply:
I knew we could add a default value to prevent the error, but with this solution we moved the rounding logic outside the Number component.  That logic was specific for mouse position components.
@offtherailz anything to add?

Reply:
I asked these changes because the rounding strategy is a coordinates (in degrees format) specific problem, and so it should not involve all the I18N utilities for all numbers. It was a wrong solution by itself, if we put apart the specific bug.
Also the component was not tested properly in the original  PR.